kube_log_watcher/main.py

```diff
@@ -231,7 +231,7 @@ def get_new_containers_log_targets(
 
             kwargs['application'] = pod_labels.get(APP_LABEL)
             kwargs['environment'] = pod_labels.get(ENVIRONMENT_LABEL, CLUSTER_ENVIRONMENT)
-            kwargs['version'] = pod_labels.get(VERSION_LABEL, 'none')
+            kwargs['version'] = pod_labels.get(VERSION_LABEL, '')
```

Comment:
Why is it better to remove the `none` default value here and then have to add it in 3 other places?

Reply:
It is rather strange default value. For all output targets it could/should be different — "none" for `symlinker` and `None` or no attribute for scalyr

Reply:
If we could not set the field at all for Scalyr, that would make sense. But with this change we're just moving the same default value to 3 different places.

Reply:
It was a first step